src/3d/shaders.qrc

```diff
@@ -24,17 +23,15 @@
         <file>shaders/preview.vert</file>
         <file>shaders/preview.frag</file>
         <file>shaders/phongDataDefined.vert</file>
-        <file>shaders/phongDataDefined.frag</file>
-        <file>shaders/goochDataDefined.frag</file>
```

Comment:
Looks like this file needs deleting too

Reply:
Ah, missed that there's a rename here -- I was looking for the removal of the goochDataDefined file